browser/BUILD.gn

```diff
@@ -133,8 +126,6 @@ source_set("browser_process") {
     "//brave/components/brave_component_updater/browser",
     "//brave/components/brave_drm",
     "//brave/components/brave_together/browser",
```

Comment:
actually also noticed this is missing `"//brave/components/brave_referrals/buildflags",`

Reply:
yes, right. I add it to 337 lines. Is it fine?

Reply:
no, the buildflag dep (also here https://github.com/brave/brave-core/pull/6297/files#diff-84f8700538180eec90bb2f4c9dff4da4R481) explicitly cannot be inside `enable_brave_referrals` because it needs to be available whether the buildflag is enabled or not

Reply:
Ah, right. Fixed it again.